rtv/packages/praw/__init__.py

```diff
@@ -358,7 +358,7 @@ def _req_error(*_, **__):
                 self.http.proxies['http'] = self.config.http_proxy
             if self.config.https_proxy:
                 self.http.proxies['https'] = self.config.https_proxy
-        self.modhash = None
+        self.modhash = ''
```

Comment:
Okay. I'll send those changes upstream. I wasn't sure if the older version would be accepting changes.